src/black/mode.py

```diff
@@ -124,6 +125,12 @@ def supports_feature(target_versions: Set[TargetVersion], feature: Feature) -> b
 class Preview(Enum):
     """Individual preview style features."""
 
+    string_processing = auto
```

Comment:
```suggestion
    string_processing = auto()
```
Unless it's even more magical than I thought